knox/knox.sh

```diff
@@ -128,8 +160,31 @@ function initialize_crontab() {
 function install() {
   local master_name
   master_name=$(/usr/share/google/get_metadata_value attributes/dataproc-master)
-  if [[ "${master_name}" == $(hostname) ]]; then
+  if [[ "${DATAPROC_IMAGE_VERSION}" == "2.2" ]]; then
+      if [[ "${master_name}" == $(hostname -A | tr -d '[:space:]' | awk -F "." '{print $1}') ]]; then
+         [[ -z "${KNOX_GW_CONFIG_GCS}" ]] && err "Metadata knox-gw-config is not provided. knox-gw-config stores the configuration files for knox."
+          
+          retry_command "apt-get update"
+          install_dependencies
+          update
+          enable_demo_ldap_for_dev_testing
+          get_config_parameters
+          replace_the_master_key
+          generate_or_replace_certificate
+          initialize_crontab
+
+         [[ -f "/lib/systemd/system/knoxldapdemo.service" ]] && systemctl restart knoxldapdemo
```

Comment:
Please write this as an `if ; then ; fi` statement

Reply:
Implemented as per feedback